Example/R87AttributedString/ViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  R87AttributedStringSwift
+//
+//  Created by Bruno Muniz Azevedo Filho on 12/11/17.
+//  Copyright © 2017 bmaf. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+	@IBOutlet weak var label: UILabel!
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		let attributedString = NSMutableAttributedString.init(string: "*Bruno* #Muniz# $Aze-ve-do$")
```

Comment:
You should use the original `TermsOfUse.txt` JSON file in the example project, it shows a real use case. You can check how the original example looked like in the `Screenshots` folder